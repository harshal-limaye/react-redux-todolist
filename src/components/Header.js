import React from 'react';
import FilterOptions from '../containers/FilterOptions';

class Header extends React.Component {
    render() {
        return (
            <div className="todo-header">
                <h1>toDoList</h1>
                <FilterOptions />
            </div>
        );
    }
}

export default Header;