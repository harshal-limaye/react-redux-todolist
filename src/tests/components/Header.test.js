import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header';

describe('should test header component', () => {
    test('should render component correctly', () => {
        const wrapper = shallow(<Header />);
        expect(wrapper).toMatchSnapshot();
    });
});