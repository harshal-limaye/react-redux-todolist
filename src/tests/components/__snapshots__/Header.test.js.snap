// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test header component should render component correctly 1`] = `
<div
  className="todo-header"
>
  <h1>
    toDoList
  </h1>
  <Connect(Filter) />
</div>
`;
